docs/other/install-gwms-frontend.md

```diff
@@ -422,7 +439,7 @@ and renew the %GREEN%pilot proxies%ENDCOLOR% and %GREEN%VO Frontend proxy%ENDCOL
             fqan = %RED%<VOMS ATTRIBUTE>%ENDCOLOR%
 
         !!! warning
-            Due to the [retirement of VOMS Admin server](https://opensciencegrid.org/technology/policy/voms-admin-retire/)
+            Due to the [retirement of VOMS Admin server](https://opensciencegrid.github.io/technology/policy/voms-admin-retire/)
```

Comment:
Can you revert this specific change? The `opensciencegrid.org` version works fine